src/components/Calculator.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable max-classes-per-file */
+import React, { Component } from 'react';
+
+class Screen extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="item screen">
+        <p>{this.props.counter}</p>
+      </div>
+    );
+  }
+}
+
+Screen.defaultProps = { counter: 0 };
```

Comment:
- [ ] Please use only one class per component you can do the output as a div inside the Calculator class component.